src/avram/delete_operation_template.cr

```diff
@@ -0,0 +1,18 @@
+class Avram::DeleteOperationTemplate
+  macro setup(type, columns, *args, **named_args)
+
+    # This makes it easy for plugins and extensions to use the base SaveOperation
+    def base_query_class : ::{{ type }}::BaseQuery.class
+      ::{{ type }}::BaseQuery
+    end
```

Comment:
```suggestion
```

Duplicate of https://github.com/luckyframework/avram/blob/be876ef87ae120650ce27b93f586ba314570663a/src/avram/save_operation_template.cr#L4-L7